cmd/svcat/instance/deprovision_cmd.go

```diff
@@ -53,6 +63,14 @@ func (c *deprovisonCmd) Validate(args []string) error {
 	}
 	c.instanceName = args[0]
 
+	if c.wait && c.rawTimeout != "-1" {
```

Comment:
This type of logic has been repeated a few times, I am wondering if you could make a helper function and pass in an anon function to do the custom work and delegate the time logic to a common place.